README.md

```diff
@@ -87,12 +82,10 @@ Aerospike.connect(config)
 
 ## Prerequisites
 
-The client is compatible with Node.js 10, 12 (LTS), 14 (LTS), 16 (LTS) and 18 (LTS). It
-supports the following operating systems: CentOS/RHEL 7/8, Debian 8/9/10/11,
-Ubuntu 18.04/20.04, as well as many Linux distributions compatible with one of
-these OS releases. macOS is also supported. The client port to Windows is a
-community supported project and suitable for application prototyping and
-development.
+The client is compatible with Node.js 19, 18 (LTS), 16 (LTS) and 14 (LTS). It
+supports the following operating systems: RHEL 7/8/9, Debian 10/11,
+Ubuntu 18.04/20.04/22.04(Focal, Jammy, Bionic), as well as many Linux distributions compatible with one of
+these OS releases. macOS versions 11/12/13 are also supported.
```

Comment:
```suggestion
The client is compatible with Node.js 19, 18 (LTS), 16 (LTS) and 14 (LTS).
It supports the following operating systems:
- RHEL 7/8/9
- Debian 10/11
- Ubuntu 18.04/20.04/22.04 (Focal, Jammy, Bionic)
- Many Linux distributions compatible with one of the above OS releases.
- macOS versions 11/12/13 are also supported.
```
Formatting these into a list makes it easier to read

Reply:
@DomPeliniAerospike I thought that macOS is not compatible with with Node.js 14?  If so you need to mention that here.